{"name": "用例标题：用户名填写重复，提示用户名已存在", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == (2 + 1)\n +  where 2 = int(2)\n +  and   2 = int(2)", "trace": "self = <testcase.test_set.TestUser object at 0x000001FC39FB9460>\ntest_input = {'cnName': '用户2', 'confirm': 'Qq123456', 'password': 'UXExMjM0NTY=', 'phoneNumber': '13200000002', ...}\nexpected = {'message': '用户名已存在！', 'status': 500}, title = '用户名填写重复，提示用户名已存在'\nyt_headers = {'authorization': 'Bearer dd4b9d3a-6418-476e-a5b0-873f124ab32c', 'content-type': 'application/json'}\n\n    @allure.story(\"新增用户信息接口\")\n    @allure.title(\"用例标题：{title}\")\n    @pytest.mark.parametrize(\"test_input, expected, title\", test_data['set']['user_data'])\n    def test_user_add(self, test_input, expected, title, yt_headers):\n    \n        count_before = self.sql_exec.select_one(self.sql['user']['count'])[0]\n        self.log.info(\"新增子账号前，查询sql数据结果为{}\".format(count_before))\n    \n    \n        response = requests.post(url=self.url_data['user']['add_user'],\n                                 headers = yt_headers,\n                                 json=test_input)\n        self.log.info(\"接口响应数据结果为{}\".format(response.json()))\n    \n        count_after = self.sql_exec.select_one(self.sql['user']['count'])[0]\n        self.log.info(\"新增子账号后，查询sql数据结果为{}\".format(count_before))\n    \n        assert response.json()['message'] == expected['message']\n        assert response.json()['status'] == expected['status']\n>       assert int(count_after) == int(count_before) + 1\nE       assert 2 == (2 + 1)\nE        +  where 2 = int(2)\nE        +  and   2 = int(2)\n\ntestcase\\test_set.py:315: AssertionError"}, "attachments": [{"name": "log", "source": "4c537111-f341-4f1d-b9d3-ebec4740b9be-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "f90474e6-90cb-43a0-95f0-a05018ecfcb2-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_input", "value": "{'username': 'SUE_ZK04_A02', 'password': 'UXExMjM0NTY=', 'confirm': 'Qq123456', 'cnName': '用户2', 'roleId': 138, 'phoneNumber': '13200000002'}"}, {"name": "expected", "value": "{'message': '用户名已存在！', 'status': 500}"}, {"name": "title", "value": "'用户名填写重复，提示用户名已存在'"}], "start": 1627631527570, "stop": 1627631527983, "uuid": "a0aa2f45-9e38-4e39-84e8-49cbde581fcb", "historyId": "c97964472474c4eb80381a2f0eab8f58", "testCaseId": "927452938cd34b471a14569bf7d98541", "fullName": "testcase.test_set.TestUser#test_user_add", "labels": [{"name": "feature", "value": "用户管理"}, {"name": "story", "value": "新增用户信息接口"}, {"name": "tag", "value": "@pytest.mark.usefixtures('userId')"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_set"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "DESKTOP-M0PGOTF"}, {"name": "thread", "value": "15648-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_set"}]}